azkaban-exec-server/src/main/java/azkaban/container/FlowContainer.java

```diff
@@ -760,6 +761,18 @@ private void logVersionSet(final ExecutableFlow flow) {
     }
   }
 
+  /**
+   * Log if this flow execution pod is autoscaled by VPA
+   * @param flow Executable flow.
+   */
+  private void logVPAEnabled(final ExecutableFlow flow) {
+    if (flow.isVPAEnabled()) {
+      logger.info(String.format("This flow execution pod %s is autoscaled by Azkaban. If this "
+              + "execution ends with Out-Of-Memory Killed, please reach out to Azkaban team for "
+              + "help.", flow.getExecutionId()));
+    }
+  }
```

Comment:
minor suggestion: also log if VPA is not enabled for the flow

Reply:
Thinking it should be fine. It is a user-oriented log line. If VPA is disabled, user does not need to see the line there which may confuse them